test/Mechanical/rotational.jl

```diff
@@ -98,4 +98,77 @@ end
     # sol = solve(prob, Rodas4())
 
     # Plots.plot(sol; vars=[inertia2.w, inertia3.w])
+end
+
+@testset "Stick-Slip" begin
+    function VelocityProfile(;name)
+        @named sine = Blocks.Sine(amplitude=10, frequency=0.1)
+        @named dz = Blocks.DeadZone(u_max=2)
+        @named lim = Blocks.Limiter(y_max=6)
+        @named output = Blocks.RealOutput()
+        connections = [
+            connect(sine.output, dz.input)
+            connect(dz.output, lim.input)
+            connect(lim.output, output)
+        ]
+        ODESystem(connections, t, [], []; name=name, systems=[sine, dz, lim, output])
+    end
+
+    @named fixed = Fixed()
+    @named spring = Spring(c=6.5)
+    @named damper = Damper(d=0.01)
+    @named inertia = Inertia(J=0.0001)
+    @named friction = RotationalFriction(f=0.001, tau_c=20, w_brk=0.06035, tau_brk=25)
+    @named vel_profile = VelocityProfile()
+    @named source = Speed(use_support=false)
+    @named angle_sensor = AngleSensor()
+
+    connections = [
+        connect(vel_profile.output, source.w_ref)
+        connect(source.flange, friction.flange_a)
+        connect(friction.flange_b, inertia.flange_a)
+        connect(inertia.flange_b, spring.flange_a, damper.flange_a)
+        connect(spring.flange_b, damper.flange_b, fixed.flange)
+        connect(angle_sensor.flange, inertia.flange_a)
+    ]
+
+    @named model = ODESystem(connections, t, systems=[fixed, inertia, spring, damper, vel_profile, source, friction, angle_sensor])
+    sys = structural_simplify(model)
+    prob = ODAEProblem(sys, Pair[], (0, 10.0))
+    sol = solve(prob, Rodas4())
+
+    # p1 = Plots.plot(sol; vars=[inertia.flange_a.phi, source.phi], title="Angular Position", labels=["Inertia" "Source"], ylabel="Angle in rad")
+    # p2 = Plots.plot(sol; vars=[source.w], title="Angular Velocity of Source", label="", ylabel="Angular Velocity in rad/s")
+    # Plots.plot(p1, p2, layout=(2,1))
+    # Plots.savefig("stick_slip.png")
+
+    @test sol[angle_sensor.phi.u] == sol[inertia.flange_a.phi]
```

Comment:
same here, there is a lot of code that builds an interesting system, but only a single trivial property of the solution is tested.